core/src/edu/asu/c3simulator/simulation/Product.java

```diff
@@ -0,0 +1,35 @@
+package edu.asu.c3simulator.simulation;
+
+
+public interface Product
+{
+	boolean retired = false;
+	Product getProduct();
+
+	public String getProductName();
+	public float getProductionCost();
+	public float getMaterialCost();
+	public float getDistributionCost();
+	public float getSellingPrice();
+	public int getEfficiency();
+	public int getNumSold();
+	public float getTotalWorth();
+	public String getLabor();
+	
+	//setters for chainging product names/modifying cost attributes
+	public String setProductName();
+	public void setProductionCost();
+	public void setMaterialCost();
+	public void setDistributionCost();
+	public void setSellingPrice();
+	public void setEfficiency();
+	//Change which laborer/s are currently producing the product.
+	public void setLabor();
```

Comment:
Interfaces do not need to specify "public" or "abstract" as all interface methods gain these automatically. 

Also, if Product is just a structure, eliminating the interface may be cleaner. I doubt we will need multiple implementations of product.